backend/model/model.go

```diff
@@ -2739,32 +2765,69 @@ func (obj *Alert) sendSlackAlert(ctx context.Context, db *gorm.DB, alertID int,
 		identifier = val
 	}
 
-	var previewText string
 	if obj.Type == nil {
 		if input.Group != nil {
 			obj.Type = &AlertType.ERROR
 		} else {
 			obj.Type = &AlertType.NEW_USER
 		}
 	}
+
+	previewText := getPreviewText(*obj.Type)
+
+	var headerBlockSet []slack.Block
+
+	var bodyBlockSet []slack.Block
+	var attachment *slack.Attachment
+
+	var sessionBlock *slack.TextBlockObject
+	if input.SessionSecureID == "" || input.SessionExcluded {
+		var sessionText string
+		if identifier == "" {
+			sessionText = "*Session* No recorded session"
+		} else {
+			sessionText = fmt.Sprintf("*Session* No recorded session (%s)", identifier)
+		}
+		sessionBlock = slack.NewTextBlockObject(slack.MarkdownType, sessionText, false, false)
+	} else {
+		sessionUserIdentifier := identifier
+		if sessionUserIdentifier == "" {
+			sessionUserIdentifier = "_unidentified_ user"
+		}
+
+		sessionLink := fmt.Sprintf("%s/%d/sessions/%s%s", frontendURL, obj.ProjectID, input.SessionSecureID, suffix)
+		sessionText := fmt.Sprintf("#%s (%s)", input.SessionSecureID, sessionUserIdentifier)
+		sessionBlock = slack.NewTextBlockObject(slack.MarkdownType, fmt.Sprintf("*Session* <%s|%s>", sessionLink, sessionText), false, false)
+	}
+
 	switch *obj.Type {
 	case AlertType.ERROR:
 		shortEvent := input.Group.Event
-		if len(input.Group.Event) > 50 {
-			shortEvent = input.Group.Event[:50] + "..."
+		if len(input.Group.Event) > 100 {
+			shortEvent = input.Group.Event[:100] + "..."
```

Comment:
could make this even longer? 100 (1.5 lines as in your example) still seems pretty short

Reply:
<img width="741" alt="Screenshot 2023-10-10 at 9 55 48 AM" src="https://github.com/highlight/highlight/assets/17744174/0138b164-20c1-41db-80b9-55a1e3cbc213">